ngraph_bridge/enable_variable_ops/ngraph_catalog.cc

```diff
@@ -87,8 +87,7 @@ bool NGraphCatalog::EncapOutputIndexNeedsCopy(const int& graphid,
     auto op_copy_indexes = itr->second;
     return (op_copy_indexes.find(index) != op_copy_indexes.end());
   }
-  // Should not reach here
-  return true;
+  return false;
```

Comment:
Can you comment on why the true changed to false?

Reply:
if you see the function it should return false. it is probably a bug dont remember.